gradle-with-plugin/build.gradle

```diff
@@ -11,6 +11,17 @@ project.description = 'Example Web Scripts, Actions and Behaviours.'
 
 version = '2.0-SNAPSHOT'
 
+sourceSets {
+    main {
+        java {
+            srcDir '../src/main/java'
```

Comment:
Configuring the sourcesets here deduplicates the java code, but makes it more confusing, because this is not necessary in a normal project.
Maybe you can symlink the `src` folder?

Reply:
Does that work transparently on Windows & Linux?

Reply:
I'm not sure, and I could not find a conclusive answer.
If it is not supported, then I would go for copy-pasting the source code. That way, you can easily just take one folder and run with it.

Reply:
Some relevant documentation
https://github.com/git-for-windows/git/wiki/Symbolic-Links

Reply:
Duplicated source code in each supproject